lean4-infoview/src/infoview/interactiveCode.tsx

```diff
@@ -25,8 +25,12 @@ export interface InteractiveTaggedTextProps<T> extends InteractiveTextComponentP
  * various embedded information such as `InfoTree`s and `Expr`s.
  * */
 export function InteractiveTaggedText<T>({fmt, InnerTagUi}: InteractiveTaggedTextProps<T>) {
-  if ('text' in fmt) return <>{fmt.text}</>
-  else if ('append' in fmt) return <>
+  if ('text' in fmt) {
+    if (isInaccessibleName(fmt.text))
+      return <span className="goal-inaccessible">{fmt.text}</span>
+    else
```

Comment:
```suggestion
```
This check is super flaky and will trigger on any text containing `✝`.  (Including `logInfo "this will all be gray ✝ not tested though"` for example.)

Reply:
I guess that's technically true... I don't know if this part is important enough to introduce a more robust server-side solution